examples/ICESat-2_DAAC_DataAccess_Example.ipynb

```diff
@@ -427,7 +427,7 @@
    "cell_type": "markdown",
    "source": [
     "#### Place the order\n",
-    "Then, we can send the order to NSIDC using the order_granules function. Information about the granules ordered and their status will be printed automatically as well as emailed to the address provided. Additional information on the order, including request URLs, can be viewed by setting the optional keyword input 'verbose' to True."
+    "Then, we can send the order to NSIDC using the order_granules function. Information about the granules ordered and their status will be printed automatically. Status information can also be emailed to the address provided when the `email` kwarg is set to `True`. Additional information on the order, including request URLs, can be viewed by setting the optional keyword input 'verbose' to True."
```

Comment:
Could you also modify Line 439 below to say `# region_a.order_granules(verbose=True, subset=False, email=False)`? Just so users looking at the Jupyter notebook know how to get the email notifications still.

Reply:
4a5b393